mne/report.py

```diff
@@ -1407,7 +1407,7 @@ def parse_folder(self, data_path, pattern='*.fif', n_jobs=1, mri_decim=2,
 
         Parameters
         ----------
-        data_path : str
+        data_path : str | pathlib.Path
```

Comment:
```suggestion
        data_path : str
```
So it seems in all other places of MNE we **don't** mention that we support `Path` / path-like objects, just `str`. I believe this should be changed – but in another PR.